crates/extensions/tedge_log_manager/src/actor.rs

```diff
@@ -169,14 +149,14 @@ impl LogManagerActor {
     /// - sends request successful (mqtt)
     async fn execute_logfile_request_operation(
         &mut self,
-        request: &LogRequestPayload,
+        request: &LogUploadCmdPayload,
     ) -> Result<(), LogManagementError> {
         let log_content = log_manager::new_read_logs(
             &self.plugin_config.files,
-            &request.log.log_type,
-            request.log.date_from,
-            request.log.lines,
-            &request.log.search_text,
+            &request.log_type,
+            request.date_from,
+            request.lines.clone(),
```

Comment:
```suggestion
            request.lines,
```
Clone is redundant as `usize` type implements `Copy` trait

Reply:
Replaced with `to_owned()`, making clippy happy.